nemo/collections/nlp/modules/common/text_generation_utils.py

```diff
@@ -889,6 +889,11 @@ def switch(val1, val2, boolean):
     return (1 - boolean) * val1 + boolean * val2
 
 
+def ceil_to_mult(n, m):
```

Comment:
Can we have a more descriptive name ? Or a short description of what this does ?

Reply:
it's a one line function, so I feel it's self documented.

If you have specific name or description you'd like added please let me know.

Thanks.